src/shogun/ensemble/MeanRule.cpp

```diff
@@ -35,7 +38,7 @@ SGVector<float64_t> CMeanRule::combine(const SGMatrix<float64_t>& ensemble_resul
 	SGVector<float64_t> mean_labels(row_sum, ensemble_result.num_rows);
 
 	float64_t scale = 1/(float64_t)ensemble_result.num_cols;
-	mean_labels.scale(scale);
+	linalg::scale<linalg::Backend::NATIVE>(mean_labels, scale);
```

Comment:
you have already written `using namespace linalg`, so you can omit the `linalg::` here I suppose.